### ISSUE TITLE
Vlang Support

### ISSUE THREAD

**Issue by ILoveAmphetamines** [hugolgst/vimsence#56]
Hello! There is a new language called Vlang that should be added if possible, please :)

**Comment by simvux** [comment]
I don't think this plugin needs to add support for individual languages for them to work.

But just as a heads up,
The V project is borderline fraudulent and should not be taken seriously. It's popularity grew off complete lies and anyone experienced in compiler development understand that it's current implementation is not going anywhere. I encourage you to look into it more than just reading the official claims about the project.
